policy_agent.py

```diff
@@ -106,6 +112,15 @@ def get_handler(self, resource_type, event_type):
         _log.debug("Looking up handler for event: %s", key)
         return self._handlers[key]
 
+    def load_cache(self):
+        """
+        Loads cache from etcd.
+        """
+        endpoints = self._client.get_endpoints()
```

Comment:
Good point.  We don't have a hostname here, so we just need to get all and filter.

Same for above - we don't have a hostname available.